libs/core/src/lib/panel/panel-content/panel-content.directive.ts

```diff
@@ -53,7 +53,4 @@ export class PanelContentDirective {
     @HostBinding('attr.id')
     id: string = 'fd-panel-content-' + panelContentUniqueId++;
 
-    /** Tabindex of the content section. */
-    @HostBinding('attr.tabindex')
-    tabindex = 0;
```

Comment:
Why did you remove this? The idea of having it the first place was that these elements (section or header) can get some interaction and then they will need the focus.

Reply:
@InnaAtanasova 
I understand but this was not the right thing to do, I think. 
- non-interactive elements should not get focus [or have tabindex](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex#accessibility_concerns): the panel content and the panel title are clearly visible and do not need to be read out by the screen reader by explicitly focusing on it or by adding tabindex. For JAWS, a11y team tests using something called virtual PC cursor mode in JAWS specifically for reading out static content like this.
- Adding tabindex with an invisible focus line also has the side effect of adding these non-interactive elements to the a11y/navigation tree (which is what the first two issues in #6176 talk about). Even [now](https://fundamental-ngx.netlify.app/#/core/panel), after tabbing to the expand/collapse button, it takes two tabs to reach `Show Code`; this shouldn't happen and can confuse the user.
- the [issue](https://github.com/SAP/fundamental-ngx/issues/3494) for which this change was added(pr https://github.com/SAP/fundamental-ngx/pull/4555) talks about title not being read when focus was on the expand/collapse arrow. We could have fixed this by using aria-labelledby instead of adding tabindex.

Reply:
@kavya-b thanks for explaining it. I was just confused why we are now removing smth that was added to fix an a11y issue :)